setup.py

```diff
@@ -14,7 +14,7 @@
 
 setup(
     name='kin',
-    version='2.1.6',
+    version='2.1.7',
```

Comment:
This is my fault, I had a non-pushed version change in local and forgot to push it to main, it shouldn't affect anything though